stdlib/public/Bionic/Glibc.swift

```diff
@@ -0,0 +1,215 @@
+//===----------------------------------------------------------------------===//
```

Comment:
Yes, I'll take another look at `stdlib/public/Glibc/CMakeLists.txt`. I think this should be possible. The difficult part is that since we're building for both Linux and Android at once, I'll need to use variables other than `CMAKE_SYSTEM_NAME` to determine which modulemap to copy. I'll try working on it some more, thanks!